kratos/utilities/intersection_utilities.h

```diff
@@ -833,17 +663,33 @@ class KRATOS_API(KRATOS_CORE) IntersectionUtilities
      * @param b Second vector
      * @return The 2D cross product value
      */
-
-    static inline double CrossProd2D(const array_1d<double,3> &a, const array_1d<double,3> &b){
+    static inline double CrossProd2D(
+        const array_1d<double,3>& a, 
+        const array_1d<double,3>& b
+        )
+    {
         return (a(0)*b(1) - a(1)*b(0));
```

Comment:
```suggestion
        const array_1d<double,3>& rA, 
        const array_1d<double,3>& rB)
    {
        return (rA(0)*rB(1) - rA(1)*rB(0));
```
If we are to change these things lets do everything.